crates/biome_css_analyze/tests/specs/nursery/useLayeredStyles/valid.css

```
@@ -0,0 +1,107 @@
+
+	50% {
+		transform: rotate(180deg);
+	}
+
+	to {
+		transform: rotate(360deg);
+	}
+}
+
+@font-face {
+	font-family: "Test";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Test" (font-family-name-quotes)

(font-family-name-quotes)